src/components/SkillPage/SkillListing.js

```diff
@@ -594,6 +603,50 @@ class SkillListing extends Component {
     this.setState({ showAuthorSkills: false });
   };
 
+  handleReportToggle = () => {
+    this.setState({
+      showReportDialog: !this.state.showReportDialog,
+    });
+  };
+
+  handleReportSubmit = () => {
+    let reportUrl = `${urls.API_URL}/cms/reportSkill.json?model=${
+      this.state.skillModel
+    }&group=${this.state.skillGroup}&language=${
+      this.state.skillLanguage
+    }&skill=${this.state.skill_name}&feedback=${
```

Comment:
Use `skill_tag` of the skill metadata object for skill name. Due to which, it gives error, when I report the skill twice.

Reply:
Still throws an error when skill tag is used.